app/controllers/kudos_controller.rb

```diff
@@ -71,6 +71,23 @@ def create
     end
   end
 
+  def update
```

Comment:
Can this be made more declarative using [strong parameters](http://edgeguides.rubyonrails.org/action_controller_overview.html#strong-parameters) and mass assignments (and doing the permission check declaratively, as part of the parameters)?